### DISCUSSION TITLE
Symlinks remaining after subscription manager removed

### DISCUSSION THREAD

**Discussion by djach7** [candlepin/subscription-manager#3394] [General]
Hello, I just had an issue with removing subscription manager and wanted to post here for more insight as to what happened. My situation was the following:
- Fedora 37 system with subscription-manager installed
- Attempted to start podman container using ubi repos, but rhel repos kept overriding
```
STEP 8/15: RUN dnf install -y gcc                    python3-devel  && dnf clean all -y
Updating Subscription Management repositories.
Unable to read consumer identity
subscription-manager is operating in container mode.

This system is not registered with an entitlement server. You can use subscription-manager to register.

Red Hat Enterprise Linux 9 for x86_64 - AppStre 0.0  B/s |   0  B     00:00    
Errors during downloading metadata for repository 'rhel-9-for-x86_64-appstream-rpms':
  - Curl error (77): Problem with the SSL CA cert (path? access rights?) for https://cdn.redhat.com/content/dist/rhel9/9/x86_64/appstream/os/repodata/repomd.xml [error setting certificate file: %(ca_cert_dir)sredhat-uep.pem]
Error: Failed to download metadata for repo 'rhel-9-for-x86_64-appstream-rpms': Cannot download repomd.xml: Curl error (77): Problem with the SSL CA cert (path? access rights?) for https://cdn.redhat.com/content/dist/rhel9/9/x86_64/appstream/os/repodata/repomd.xml [error setting certificate file: %(ca_cert_dir)sredhat-uep.pem]
Error: building at STEP "RUN dnf install -y gcc                    python3-devel  && dnf clean all -y": while running runtime: exit status 1
```
- removed subscription manager with `dnf remove subscription-manager`
- error persisted 
- found that the symlinks for `rhsm` and `etc-pki-entitlement` within `/usr/share/rhel/secrets/` were not broken by removing subscription manager
- broke symlinks
- successful podman build

This is not a dire issue and is no longer blocking me, but I am wondering if anyone knows why those symlinks within `/usr/share/rhel/secrets/` remained intact even after subscription manager was removed.

**Comment by jirihnidek** [comment]
@djach7 Can you please convert (move) this issue to GitHub issue? I just enabled issues for subscription-manager again.
